types/google-apps-script/google-apps-script.card-service.d.ts

```diff
@@ -152,11 +152,13 @@ declare namespace GoogleAppsScript {
     }
     /**
      * A builder for Card objects.
+     * https://developers.google.com/apps-script/reference/card-service/card-builder
      */
     interface CardBuilder {
       addCardAction(cardAction: CardAction): CardBuilder;
       addSection(section: CardSection): CardBuilder;
       build(): Card;
+      setFixedFooter(footer: FixedFooter): CardBuilder;
```

Comment:
```suggestion
      setFixedFooter(fixedFooter: FixedFooter): CardBuilder;
```
https://developers.google.com/apps-script/reference/card-service/card-builder#setFixedFooter(FixedFooter)

Reply:
It seems good.
@jessety Can you fix it before I approve?

Reply:
Nice catch, thanks @mtgto!